packages/build/src/tarball.ts

```diff
@@ -84,7 +123,7 @@ export const tarballWindows = (input: string, filename: string): void => {
 export type TarballFile = { path: string; contentType: string };
 
 /**
- * Create a gtarballped tarball or zip for the provided options.
```

Comment:
"tarballped" is a great word, i think it should mean something!

Reply:
ahahhaha